--- conf/config.pl.orig	2009-09-29 12:27:00.000000000 +0300
+++ conf/config.pl	2009-09-29 22:17:53.000000000 +0300
@@ -1270,6 +1270,7 @@
             '--times',
             '--block-size=2048',
             '--recursive',
+            '--specials',
 
 	    #
 	    # Rsync >= 2.6.3 supports the --checksum-seed option
@@ -1309,6 +1310,7 @@
 	    '--relative',
 	    '--ignore-times',
 	    '--recursive',
+	    '--specials',
 
 	    #
 	    # Rsync >= 2.6.3 supports the --checksum-seed option
@@ -1920,7 +1922,7 @@
 #    --> administrative users are only craig and celia'.
 #
 $Conf{CgiAdminUserGroup} = '';
-$Conf{CgiAdminUsers}     = '';
+$Conf{CgiAdminUsers}     = 'backuppc';
 
 #
 # URL of the BackupPC_Admin CGI script.  Used for email messages.
@@ -1967,7 +1969,7 @@
 # dates (MM/DD), a value of 2 uses full YYYY-MM-DD format, and zero
 # for international dates (DD/MM).
 #
-$Conf{CgiDateFormatMMDD} = 1;
+$Conf{CgiDateFormatMMDD} = 2;
 
 #
 # If set, the complete list of hosts appears in the left navigation
