--- configure.pl.orig	2009-09-29 01:52:58.000000000 +0300
+++ configure.pl	2009-09-29 02:12:16.000000000 +0300
@@ -1,4 +1,4 @@
-#!/bin/perl
+#!/usr/bin/perl
 #============================================================= -*-perl-*-
 #
 # configure.pl: Configuration and installation program for BackupPC
@@ -157,7 +157,7 @@
 # config file to get all the defaults.
 #
 my $ConfigPath = "";
-my $ConfigFileOK = 1;
+my $ConfigFileOK = 0;
 while ( 1 ) {
     if ( $ConfigFileOK && -f "/etc/BackupPC/config.pl" ) {
         $ConfigPath = "/etc/BackupPC/config.pl";
@@ -206,6 +206,11 @@
             if ( !($bpc = BackupPC::Lib->new(".", ".", $confDir, 1)) );
     %Conf = $bpc->Conf();
     %OrigConf = %Conf;
+
+    # Let's hardcode the config path so the script doesn't get confused
+    $ConfigPath = "/etc/BackupPC/config.pl";
+    $opts{"config-dir"} = $ConfigPath;
+
     if ( !$opts{fhs} ) {
         ($Conf{TopDir} = $ConfigPath) =~ s{/[^/]+/[^/]+$}{}
                     if ( $Conf{TopDir} eq '' );
